lib/question.js

```diff
@@ -1192,20 +1192,17 @@ module.exports = {
           }
           callback(null);
         },
-        (callback) => {
-          sqldb.query(sql.select_issues_for_variant, { variant_id: variant.id }, (err, result) => {
-            if (ERR(err, () => {})) {
-              return callback(err);
-            }
-
-            if (result.rowCount > 0) {
-              success = false;
-              logger.verbose(`ERROR: ${result.rowCount} issues encountered during test.`);
-            } else {
-              logger.verbose('Success: no issues during test');
-            }
-            callback(null);
+        async () => {
+          const result = await sqldb.queryOneRowAsync(sql.select_issue_count_for_variant, {
```

Comment:
Ugh, I can't believe we were fetching them all just to get `rowCount`. I'm not going to `git blame` because I'm sure it's me 🤦